eslint-default.js

```diff
@@ -4,42 +4,45 @@ module.exports = {
     project: ["tsconfig.json"],
     ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
     sourceType: "module", // Allows for the use of imports
-    ecmaFeatures: {
-      jsx: true, // Allows for the parsing of JSX
-    },
   },
-  plugins: ["@typescript-eslint", "sonarjs"],
+  plugins: ["@typescript-eslint", "sonarjs", "unused-imports"],
   extends: [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:@typescript-eslint/recommended-requiring-type-checking",
-    "plugin:react/recommended",
-    "plugin:sonarjs/recommended",
     "plugin:eslint-comments/recommended",
-    "prettier", // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
+    "plugin:jest/recommended",
```

Comment:
Nebylo by lepší použít https://www.npmjs.com/package/eslint-plugin-vitest, když i pro test runner preferujeme vitest?

Reply:
A používáme to na všech projektech? Vlastně si nejsem jistý, jestli by tu vůbec mělo být jedno z toho. Pokud opravdu na všech projektech používáme stejný runner, tak ano. Na NTT máme jest. Ale vzhledem k tomu, kolik je tam testů, nebude problém přejít na vitest. Asi bych to otevřel na fe session.

Reply:
V eslint-react bude vitest, v eslint-node bude jest.